CHANGELOG.md

```diff
@@ -17,3 +17,12 @@
 ## 0.1.1
 
 - Updated flutter_form version to 2.0.1
+
+## 0.2.0
```

Comment:
Add this to the top of the file so the latest version is always on top.

Reply:
fixed